php/class-wp-seo-settings.php

```diff
@@ -383,6 +382,7 @@ public function example_permalink( $section ) {
 	 */
 	public function example_term_archive( $section ) {
 		if ( $term = get_terms( str_replace( 'archive_', '', $section['id'] ), array( 'number' => 1 ) ) ) {
+			// $term is a \WP_Term, which wpcom_vip_get_term_link() allows.
```

Comment:
I agree, this comment is confusing. I'll remove it. It was extra detail about why I disabled the `get_term_link()` sniff: https://github.com/alleyinteractive/wp-seo/pull/58/files/e461cc47602ce84801e7a0438b315c4c35c80ae5#diff-115b462d2cf48554cb097171b6192e80R10.